src/v2/guide/events.md

```diff
@@ -252,27 +256,15 @@ Here's the full list of key modifier aliases:
 - `.left`
 - `.right`
 
+<p class="tip">A few keys (`.esc` and all arrow keys) have inconsistent `key` values in IE9, so these built-in aliases should be preferred if you need to support IE9.</p>
+
 You can also [define custom key modifier aliases](../api/#keyCodes) via the global `config.keyCodes` object:
 
 ``` js
 // enable `v-on:keyup.f1`
 Vue.config.keyCodes.f1 = 112
 ```
 
-### Automatic Key Modifiers
-
-> New in 2.5.0+
-
-You can also directly use any valid key names exposed via [`KeyboardEvent.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) as modifiers by converting them to kebab-case:
-
-``` html
-<input @keyup.page-down="onPageDown">
-```
```

Comment:
I guess the key `PageDown` comes from here. Maybe we could rearrange these paragraphs to preserve this code example to show how kebab-case name converted. I will leave more advice for this to PR https://github.com/vuejs/vuejs.org/pull/1970. @phanan 😉